Frameworks/Foundation/NSSet.mm

```diff
@@ -554,12 +573,93 @@ - (void)setValue:(id)value forKey:(NSString*)key {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (NSString*)descriptionWithLocale:(id)locale {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return [self descriptionWithLocale:locale indent:0];
+}
+
+/**
+ @Status Interoperable
+*/
+- (NSString*)description {
+    return [self descriptionWithLocale:nil indent:0];
+}
+
+- (NSString*)descriptionWithLocale:(id)locale indent:(NSUInteger)level {
```

Comment:
I'm working on this w/ a few other changes in NSDictionary so I'll clean these all up in a separate PR once I've got them working as expected w/ tests n everything